src/Microsoft/Silk.NET.DXGI/DXGI.cs

```diff
@@ -28,6 +28,10 @@ public override bool IsExtensionPresent(string extension)
         {
             throw new NotImplementedException();
         }
+
+        [NativeName("Type", "unsigned int")]
+        [NativeName("Name", "DXGI_CREATE_FACTORY_DEBUG")]
+        public const uint CreateFactoryDebug = unchecked((uint) 0x1);
```

Comment:
Can constants be at the top of the class please (above everything else, including the static method)? Thanks!

Reply:
sure thing